documentation/biblatex-manuscripts-philology.tex

```diff
@@ -65,26 +71,41 @@ \subsection{Mandatory}
 
 \begin{fieldlist}
 
-\fielditem{collection}{literal} the collection in the library. For example: \verb+Supplément grec+.
+\fielditem{collection}{literal} the collection in the library. For example: 
+\verb+Supplément grec+ or \verb+Vaidyaka+.
 
-\fielditem{location}{literal} the city or place where the manuscript is kept. For example: \verb+Paris+ or \verb+Oxford+.
+\fielditem{location}{literal} the city or place where the manuscript is kept. For 
+example: \verb+Paris+ or \verb+Oxford+ or \verb+Varanasi+.
 
-\fielditem{library}{literal} the library where the manuscript is kept. For example: \verb+Bibliothèque Nationale de France+.
+\fielditem{library}{literal} the library where the manuscript is kept. For example: 
+\verb+Bibliothèque Nationale de France+ or \verb+Sarasvatī Bhavan Library+.
```

Comment:
We should correct to "Bibliothèque nationale de France"

Reply:
I agree.